### PR TITLE
Review page

### PR DESCRIPTION
**Pull request by andyhd**
Not merging yet. Currently removing null fields and completed fields that are obsolete.

### CODE REVIEW

**Comment by s-block** [comment]
looks good

**Comment by andyhd** [comment]
Not merging yet.

**Comment by tyom** [comment]
Some issues on Property section on smaller screens. Doesn't look great with two properties, I imagine it's pretty bad with three. Although it may not be an issue right now.

screen shot 2015-02-16 at 15 47 02

Also, there are some some issues with values on review page, e.g. `£None -- Please select --` and `£0 per week`. 

screen shot 2015-02-16 at 15 50 20

**Comment by andyhd** [comment]
Yeah, I'm still working on the null values as I said above. What's wrong with £0 per week? It should be £0.00 per week?

**Comment by tyom** [comment]
> What's wrong with £0 per week? It should be £0.00 per week?

I think it should just be 0. “Per week” is redundant there.

**Comment by andyhd** [comment]
@jenkins-moj retest this please

**Comment by s-block** [comment]
retest this please

**Comment by s-block** [comment]
retest this please

File: cla_public/apps/checker/means_test.py [hunk 1]
```diff
@@ -0,0 +1,596 @@
+# -*- coding: utf-8 -*-
+"Means Test class"
+
+from collections import Mapping
+from copy import deepcopy
+import logging
+from pprint import pformat
```
Comment (andyhd) [inline cla_public/apps/checker/means_test.py]: don't want this - debug output

**Comment by andyhd** [comment]
Ok, looks good to me :shipit:

**Comment by andyhd** [comment]
retest this please

**Comment by andyhd** [comment]
retest this please

**Pull request merged**